### ISSUE TITLE
Loss in paper in code

### ISSUE THREAD

**Issue by Sukaiqi** [wuyuebupt/doubleheadsrcnn#1]
Hello,I did not see the loss in the paper in the code.Can you point me to this place？Thank you very much！
For example： L=w^fc L^fc+w^conv L^conv+L^rpn
`        loss_classifier, loss_box_reg = self.loss_evaluator(
            [class_logits], [box_regression]
        )
        loss_classifier_fc, loss_box_reg_fc = self.loss_evaluator(
            [class_logits_fc], [box_regression_fc]
        )
`

**Comment by wuyuebupt** [comment]
Hi, thanks for interested in our work. The loss weights multiplied to the loss for roi heads are from line 83 to 86 in the file maskrcnn_benchmark/modeling/roi_heads/box_head/box_head.py. The rpn loss is added at maskrcnn_benchmark/engine/trainer.py#L97.

**Comment by Sukaiqi** [comment]
OK, thanks for your answers. I find it. But I still have a question.Can you help me answer？Thank you very much！
 L=w^fc L^fc+w^conv L^conv+L^rpn in Double-Head(not Double-Head-Ext)，L^fc=L_cls^fc+L_reg^fc or L^fc = L_cls^fc ?

**Comment by wuyuebupt** [comment]
If double-head (not double-head-ext), there will not be a regression loss for fc head. So it will be the L^fc = L_cls^fc.

**Comment by Sukaiqi** [comment]
Thank you very much！
